Source/WebCore/rendering/PositionedLayoutConstraints.cpp

```diff
@@ -397,23 +396,60 @@ void PositionedLayoutConstraints::computeBlockStaticDistance(const RenderBox& re
         m_insetBefore.setValue(LengthType::Fixed, staticLogicalTop);
 }
 
+void PositionedLayoutConstraints::fixupLogicalLeftPosition(RenderBox::LogicalExtentComputedValues& computedValues) const
+{
+    if (m_writingMode.isHorizontal()) {
+        CheckedPtr container = dynamicDowncast<RenderBox>(container());
+        if (container->shouldPlaceVerticalScrollbarOnLeft())
+            computedValues.m_position += container->verticalScrollbarWidth();
```

Comment:
```
if (CheckedPtr container = dynamicDowncast<RenderBox>(container(); container && container->shouldPlaceVerticalScrollbarOnLeft())
    computedValues.m_position += container->verticalScrollbarWidth();
```
gives better scoping but I don't feel too strongly in this case as `container` is already inside `if { }`.